styles.css

```diff
@@ -374,27 +404,38 @@ html {
     width: 100%;
     box-shadow: rgba(0, 0, 0, 0.1) 0 4px 12px;
     z-index: 6;
+    margin: 0 auto;
+  }
+
+  .info {
+    /* max-width: 1200px; */
```

Comment:
- [ ] kindly consider removing all the commented pice of code in your code base because they make your project look messy